### ISSUE TITLE
Inappropriate value for attribute "route" - "ipv6_cidr_block" is required

### ISSUE THREAD

**Issue by rossigee** [hashicorp/terraform-provider-aws#13854]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
$ terraform -v
Terraform v0.12.24
+ provider.aws v2.67.0
+ provider.cloudflare v2.7.0
+ provider.dns v2.2.0
+ provider.kubernetes v1.11.3
+ provider.local v1.4.0
+ provider.null v2.1.2
+ provider.rancher2 v1.8.3
+ provider.random v2.2.1
+ provider.template v2.1.2
+ provider.vault v2.11.0
```

### Affected Resource(s)

* aws_route_table

### Terraform Configuration Files

```hcl
resource "aws_route_table" "hmviart" {
  count = var.use_aws

  vpc_id = aws_vpc.hmviart[0].id

  route = [
    {
      cidr_block                = "0.0.0.0/0"
      egress_only_gateway_id    = ""
      gateway_id                = aws_internet_gateway.hmviart[0].id
      instance_id               = ""
      #ipv6_cidr_block           = ""
      nat_gateway_id            = ""
      network_interface_id      = ""
      transit_gateway_id        = ""
      vpc_peering_connection_id = ""
    },
  ]
}
```
### Debug Output

With the 'ipv6_cidr_block' commented out...

```
Inappropriate value for attribute "route": element 0: attribute
"ipv6_cidr_block" is required.
```
Uncommented, with a blank string provided (which was working fine before I upgraded the provider)...

```
Error: expected "route.0.ipv6_cidr_block" to be a valid IPv4 Value, got : invalid CIDR address: 

  on modules/aws/network.tf line 45, in resource "aws_route_table" "hmviart":
  45: resource "aws_route_table" "hmviart" {
```

### Expected Behavior

The `ipv6_cidr_block` attribute should be *Optional* as per docs...

https://www.terraform.io/docs/providers/aws/r/route_table.html#ipv6_cidr_block

### Actual Behavior

As per message in Debug Output, not only is this attribute no longer optional, but it also appears to expect an IPV4 address for a field that you would expect to take an IPV6 subnet value.

### Steps to Reproduce

1. `terraform apply`

**Comment by ewbankkit** [comment]
Related:
* #13003
* #13778

**Comment by ewbankkit** [comment]
This was addressed via https://github.com/hashicorp/terraform-provider-aws/pull/13778, released in **v2.68.0** of the AWS Provider.
